redaxo/src/addons/structure/plugins/content/lib/minibar_frontend.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * @package redaxo\core
+ */
+class rex_minibar_structure_content_frontend extends rex_minibar_element
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function render()
+    {
+        $article = rex_article::getCurrent();
+
+        if (!$article) {
+            return
+                '<div class="rex-minibar-group">
+                    <a href="#">
```

Comment:
Fehlt hier noch ein href?

Reply:
> Fehlt hier noch ein href?

Nein, eher müsste es in ein `span` umgewandelt werden, wenn keine Url angegeben. Betrifft auch [Zeile 41](https://github.com/redaxo/redaxo/pull/1921/files/17b3b046049e1d1c26360211d4b912f2e5e0efb6#diff-59669715e23ea7d4b292fc2ea9896d47R41)